docker-entrypoint.sh

```diff
@@ -47,4 +47,8 @@ EOT
     fi
 fi
 
+if [ -n "$PMA_PASSWORD_FILE" ]; then
+	export PMA_PASSWORD="$(cat $PMA_PASSWORD_FILE)"
```

Comment:
This implementation bypasses one of the main advantages of using docker secrets: sensitive information is not stored directly in the an environment variable. Instead, it is read from a secret file at runtime by application code. Nothing sensitive should ever be stored directly in an environment variable, as is being done here.